scripts/configure.js

```diff
@@ -0,0 +1,166 @@
+const _ = require('lodash');
+const inquirer = require('inquirer');
+const winston = require('winston');
+const request = require('request');
+const fs = require('fs');
+const path = require('path');
+
+const configFilePath = path.resolve(__dirname, '../config/core.json');
+
+const log = new winston.Logger({
+    transports: [new winston.transports.Console({
+        level: 'info',
+        handleExceptions: true,
+        colorize: true,
+    })],
+});
+
+const regex = {
+    fqdn: new RegExp(/^https?:\/\/(?:[a-zA-Z0-9-]+\.)+[a-zA-Z]{2,64}\/?$/),
+    ipv4: new RegExp(/((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.|$)){4}/),
+};
+
+const params = {
+    panelurl: {
+        name: 'Panel url',
+        cmdoption: 'p',
+        validate: value => {
+            if (value === '') {
+                return 'The panel url is required';
+            }
+            if (!value.match(regex.fqdn) && !value.match(regex.ipv4)) {
+                return 'Please provide either a URL (with protocol) or an IPv4 address.';
+            }
+            return true;
+        },
+    },
+    token: {
+        name: 'Configuration token',
+        cmdoption: 't',
+        validate: value => {
+            if (value === '') {
+                return 'A configuration token is required.';
+            }
+            if (value.length !== 32) {
+                return 'The token provided is invalid. It must be 32 characters long.';
```

Comment:
Its generally better to make this `if (value.length === 32) return true;`. This will prevent an error or some freak thing from skipping this and returning true below. I try to make things in the daemon return false by default, and only return true if they match specific rules.